__tests__/server.test.js

```diff
@@ -0,0 +1,40 @@
+const data = require("../db/data/test-data")
+const seed = require("../db/seeds/seed.js")
+const db = require("../db/connection.js")
+const request = require("supertest")
+const app = require("../app.js")
+
+beforeEach(() => {
+    return seed(data)
+})
+
+afterAll(() => {
+    db.end()
+})
+
+describe("/api/topics", () => {
+    describe("GET", () => {
+        test("200: Responds with an array of topic objects, each with a key of description and slug, whose values are both strings", () => {
+            return request(app)
+            .get("/api/topics")
+            .expect(200)
+            .then((response) => {
+                response.body.topics.forEach((topic) => {
```

Comment:
when you are using a forEach to test multiple things please test for the array of the length before. If your array is empty they the tests would be skipped but testing for the length would still run